shell/src/main/java/alluxio/cli/validation/HdfsConfParityValidationTask.java

```diff
@@ -0,0 +1,140 @@
+package alluxio.cli.validation;
+
+import alluxio.cli.ValidateUtils;
+import alluxio.conf.AlluxioConfiguration;
+import alluxio.util.io.PathUtils;
+
+import org.apache.commons.cli.Option;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Compares HDFS configuration in Alluxio and in HDFS environment variables.
+ * */
+@ApplicableUfsType(ApplicableUfsType.Type.HDFS)
+public class HdfsConfParityValidationTask extends HdfsConfValidationTask {
```

Comment:
This is the previous `alluxio.cli.validation.HdfsValidationTask` where the hdfs configs defined in Alluxio property are compared against the hadoop config defined in hadoop env variables.